### PR TITLE
Run3-alca204Z Change defaults for AlCaIsoTrackFilter and AlCaHBHEMuonFilter for test runs

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Change defaults for AlCaIsoTrackFilter and AlCaHBHEMuonFilter for test runs

#### PR validation:

It is tested with prescale factors > 1 and with chosen HLT's for HBHEMuonFilter. This setting is only for the test runs

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Not a full backport. Defaults are with different default prescale factors

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Calibration/HcalAlCaRecoProducers/plugins/AlCaHBHEMuonFilter.cc [hunk 1]
```diff
@@ -135,68 +137,79 @@ bool AlCaHBHEMuonFilter::filter(edm::Event& iEvent, edm::EventSetup const& iSetu
...
-      for (unsigned int i = 0; i < trigNames_.size(); ++i) {
-        if (triggerNames_[iHLT].find(trigNames_[i]) != std::string::npos) {
-          if (hlt > 0) {
-            ok = true;
-          }
+  if (trigNames_.empty()) {
+    ok = true;
+  } else {
+    edm::Handle<edm::TriggerResults> triggerResults;
+    iEvent.getByToken(tok_trigRes_, triggerResults);
```
Comment (tvami) [inline Calibration/HcalAlCaRecoProducers/plugins/AlCaHBHEMuonFilter.cc:146]: The preferred method is with the `getHandle`

File: Calibration/HcalAlCaRecoProducers/plugins/AlCaHBHEMuonFilter.cc [hunk 2]
```diff
@@ -135,68 +137,79 @@ bool AlCaHBHEMuonFilter::filter(edm::Event& iEvent, edm::EventSetup const& iSetu
...
-      iEvent.getByToken(tok_Muon_, _Muon);
+  }
+  if (ok) {
+    //Step2: Get geometry/B-field information
+    const MagneticField* bField = &(iSetup.getData(tok_magField_));
+    const CaloGeometry* geo = &(iSetup.getData(tok_geom_));
+
+    // Relevant blocks from iEvent
+    edm::Handle<reco::MuonCollection> _Muon;
+    iEvent.getByToken(tok_Muon_, _Muon);
```
Comment (tvami) [inline Calibration/HcalAlCaRecoProducers/plugins/AlCaHBHEMuonFilter.cc:174]: Again, please use `getHandle` instead

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by tvami** [comment]
+alca

**Comment by qliphy** [comment]
+1

**Pull request merged**